trollmoves/server.py

```diff
@@ -428,7 +428,22 @@ def __init__(self, attrs, publisher):
 
     def run(self):
         """Start listening to messages."""
-        with Subscribe('', topics=self.attrs['listen'], addr_listener=True) as sub:
+        nameserver = self.attrs.get('nameserver')
+        if nameserver in ('false', 'False'):
```

Comment:
why not use `FALSY` here?

Reply:
I'll move `FALSY` to `__init__` and use it from there where needed.

Reply:
Done